snapshot.yaml

```diff
@@ -1,16 +1,18 @@
 name: lootbox-snapshot
-resolver: lts-11.17
+resolver: lts-14.14
```

Comment:
Why not 14.15, the latest one?

Reply:
Well, because when you Google “stackage LTS” the first link is to 14.14. Of course I’ll never be bothered to go to stackage.org and try to figure out which one is actually the latest 🤷‍♂.

Reply:
You could use the second link

Reply:
Do you think I’m a computer wizard or what?